tasmota/tasmota_xx2c_global/xx2c_interface.ino

```diff
@@ -406,7 +406,15 @@ const uint8_t kI2cList[] = {
 #endif
 
 #ifdef XI2C_96
-  XI2C_96
+  XI2C_96,
+#endif
+
+#ifdef XI2C_97
+  XI2C_97,
+#endif
+
+#ifdef XI2C_98
+  XI2C_98
 #endif
```

Comment:
SAME HERE. DO NOT CHANGE xx2c_interface.ino !!!

This breaks I2CDriver command